data/korath/korath ships.txt

```diff
@@ -387,6 +387,48 @@ ship "'olofez"
 	description "	Fighters do not come equipped with a hyperdrive. You cannot carry a fighter unless you have a ship in your fleet with a fighter bay."
 
 
+
+ship "Kas'ik Tek 7"
+	sprite "ship/kas-ik tek 11"
+	thumbnail "thumbnail/kas-ik tek 11"
```

Comment:
Either you missed these ones or I can understand why you didn't want to.  If the latter though, your copyright needs to go back to 11

```suggestion
	sprite "ship/kas-ik tek 7"
	thumbnail "thumbnail/kas-ik tek 7"
```

Reply:
Oops